hack/docs/template.tmpl

```diff
@@ -7,7 +7,7 @@
 {{- $notationType = "plaintext" }}
 {{- end -}}
 {{- end -}}
-<pre class="helm-vars-default-value" language-yaml" lang="{{ $notationType }}">
```

Comment:
please revert this change everywhere as we're relying on this class in docs website

Reply:
@AndrewChubatiuk , Would `class="language-yaml"` do instead? what it now isn't a legal XML ...

Reply:
why is this important? where do you use it?

Reply:
My apologies - I wanted to suggest `helm-vars-default-value` - had a brain meltdown :( . 
Also, My CSS /XHTML is extremely hence missing options that there was an extra `"` .